packages/excalidraw/components/App.tsx

```diff
@@ -4146,10 +4146,24 @@ class App extends React.Component<AppProps, AppState> {
           | ExcalidrawArrowElement
           | undefined;
 
-        const step = elbowArrow
-          ? elbowArrow.startBinding || elbowArrow.endBinding
-            ? 0
-            : ELEMENT_TRANSLATE_AMOUNT
+        const preventKeyboardMove = selectedElements
+          .filter(isElbowArrow)
+          .find((arrow) => {
+            const startMissing =
```

Comment:
`startMissing` is misleading here IMO, it's more like `startElementNotInSelection`.

Reply:
changed it